distribution/lib/Standard/Database/0.0.0-dev/package.yaml

```diff
@@ -8,3 +8,25 @@ authors:
 maintainers:
   - name: Enso Team
     email: contact@enso.org
+component-groups:
+  extends:
+    - Standard.Base.Input:
+        exports:
+          - Standard.Database.Connection.Database.connect
+    - Standard.Base.Select:
+        exports:
+          - Standard.Database.Data.Table.Table.at
+          - Standard.Database.Data.Table.Table.select
+          - Standard.Database.Data.Table.Table.select_columns
+          - Standard.Database.Data.Table.Table.rename_columns
+    - Standard.Base.Join:
+        exports:
+          - Standard.Database.Data.Table.Table.join
+          - Standard.Database.Data.Table.Table.group
+    - Standard.Base.Transform:
+        exports:
+          - Standard.Database.Data.Table.Table.remove_columns
+          - Standard.Database.Data.Table.Table.reorder_columns
+          - Standard.Database.Data.Table.Table.sort_columns
```

Comment:
I think these 3 actually belong in `Select`. We were working on them as part of the "Select Library" and they do not transform any data, merely select which columns are in output and in what order. Essentially same job as `select_columns`.